### PR TITLE
Testnet: Fullnode address update

### PR DESCRIPTION
**Pull request by lezzokafka**
Updated the fullnode address

### CODE REVIEW

**Review by tomtau — APPROVED** [review]
thanks! was about to fix that

File: docs/getting-started/thaler-testnet.md [hunk 1]
```diff
@@ -271,8 +273,8 @@ you can skip to Step 3-b-5.
 ### Step 3-b-1. Create a staking address
 This can be done, for example, with the client-cli command-line tool. Set the required environment variables:
 
-- `CRYPTO_CHAIN_ID=testnet-thaler-crypto-com-chain-42
-- `CRYPTO_CLIENT_TENDERMINT <YOUR FULL NODE, e.g. ws://localhost:26657/websocket or ws://13.80.66.193:26657/websocket>`
+- `CRYPTO_CHAIN_ID=testnet-thaler-crypto-com-chain-42`
+- `CRYPTO_CLIENT_TENDERMINT <YOUR FULL NODE, e.g. ws://localhost:26657/websocket or ws:/13.94.208.212:26657/websocket>`
```
Comment (devashishdxt) [inline docs/getting-started/thaler-testnet.md:277]:
```suggestion
- `CRYPTO_CLIENT_TENDERMINT <YOUR FULL NODE, e.g. ws://localhost:26657/websocket or ws://13.94.208.212:26657/websocket>`
```

File: docs/getting-started/thaler-testnet.md [hunk 2]
```diff
@@ -289,7 +291,7 @@ stating who you are and your staking address (@devashishdxt or @lezzokafka would
 If you plan to test a production setting with the Tendermint Key Management System (KMS) tool,
 please see [production deployment notes](notes-on-production-deployment.md) how it can be converted at the current (0.7) version.
 In a development mode, the full key pair is located in the `priv_validator_key.json` file (generated with `tendermint init`). 
-The public key should be in the base64-encoded `R9/ktG1UifLZ6nMHNA/UZUaDiLAPWt+m9I4aujcAz44=`.
+The public key should be in the base64-encoded,e.g.`R9/ktG1UifLZ6nMHNA/UZUaDiLAPWt+m9I4aujcAz44=`.
```
Comment (devashishdxt) [inline docs/getting-started/thaler-testnet.md:294]:
```suggestion
The public key should be base64-encoded, e.g. `R9/ktG1UifLZ6nMHNA/UZUaDiLAPWt+m9I4aujcAz44=`.
```

**Review by devashishdxt — CHANGES_REQUESTED** [review]

File: docs/getting-started/thaler-testnet.md (hunk 1 above)
Comment (tomtau) [reply docs/getting-started/thaler-testnet.md:277]: good catches, I'll open another PR

**Pull request merged**